src/index.spec.ts

```diff
@@ -317,24 +320,10 @@ describe('initialization options', () => {
       getInstance = reloadedModule.getInstance;
     });
 
-    global.fetch = jest.fn(() => {
-      const ufc = returnUfc(MOCK_UFC_RESPONSE_FILE);
-
-      return Promise.resolve({
-        ok: true,
-        status: 200,
-        json: () => Promise.resolve(ufc),
-      });
-    }) as jest.Mock;
+    global.fetch = jest.fn(() => Promise.reject('Each test case should mock fetch'));
```

Comment:
Great idea

Reply:
It might be better to use `spyFn = jest.spyOn(global, 'fetch')` in this case so that you can restore the original function via `spy.mockRestore()` in an `afterAll`